leaf-ui/js/displayAnalysis.js

```diff
@@ -554,3 +571,16 @@ function getConfigHtml(id){
             '<div class="dropdown-container"></div>' +
            '</div>';
 }
+
+/**
+ * Reset global analysisRequest to default analysisRequest settings
+ * while preserving userAssignmentsList
+ */
+function defaultAnalysisRequest(){
```

Comment:
super optional: consider renaming to something like resetToDefault etc etc because calling defaultAnalysisRequest makes me feel like it should be returning something instead of updating a global variable...